subcommands/targets/edit.go

```diff
@@ -65,7 +65,7 @@ func doEdit(cmd *cobra.Command, args []string) {
 	edit.Stdin = os.Stdin
 	logrus.Debug("Running editor and waiting for it to finish...")
 	if err := edit.Run(); err != nil {
-		fmt.Println("Editing cancelled: ", err)
+		logrus.Warning("Editing cancelled: ", err)
```

Comment:
This one might be tempting, but based on the context it is also an informational message.